src/schema_tools/core.cljc

```diff
@@ -257,11 +260,11 @@
   existing extra keys if defined."
```

Comment:
The docstring still needs updating, right? It does not remove existing extra keys now.